src/services/filters.ts

```diff
@@ -45,15 +45,11 @@ export function getLabelOptions(labels: string[]) {
 
   return [{ label: 'All', value: ALL_VARIABLE_VALUE }, ...labelOptions];
 }
+// @todo remove "level" as well?
```

Comment:
Do we need this todo?

Reply:
Nope, we want to remove level and extracted_level AFAIK